src/battle_script_commands.c

```diff
@@ -1019,6 +1020,47 @@ bool32 IsMoveNotAllowedInSkyBattles(u32 move)
     return (gBattleStruct->isSkyBattle && IsMoveSkyBattleBanned(gCurrentMove));
 }
 
+static void SetSameMoveTurnValues(u32 moveEffect)
+{
+    bool32 isAnyTargetAffected = IsAnyTargetAffected();
+
+    switch (moveEffect)
+    {
+    case EFFECT_FURY_CUTTER:
+        if (!isAnyTargetAffected)
+            gDisableStructs[gBattlerAttacker].furyCutterCounter = 0;
+        else if (gDisableStructs[gBattlerAttacker].furyCutterCounter < 5)
+            gDisableStructs[gBattlerAttacker].furyCutterCounter++;
+        break;
+    case EFFECT_ROLLOUT:
+        if (!isAnyTargetAffected)
+        {
+            gDisableStructs[gBattlerAttacker].rolloutTimer = 0;
+            gBattleMons[gBattlerAttacker].volatiles.multipleTurns = FALSE;
+        }
+        else if (++gDisableStructs[gBattlerAttacker].rolloutTimer == 5)
```

Comment:
```suggestion
        if (!isAnyTargetAffected || ++gDisableStructs[gBattlerAttacker].rolloutTimer == 5)
```
Same outcome from both `if`s